packages/rocketchat-file-upload/lib/FileUpload.js

```diff
@@ -56,5 +57,9 @@ FileUpload = {
 };
 
 RocketChat.settings.get('FileUpload_MaxFileSize', function(key, value) {
-	maxFileSize = value;
+	try {
+		maxFileSize = parseInt(value);
+	} catch (e) {
+		maxFileSize = 2097152; // the default value
```

Comment:
Is there anyway to maybe fetch that?  Do we want to have this value hard coded here?

Reply:
This value is returned when the setting is updated. Should they manage to set an invalid integer as the value of that setting (which shouldn't happen in theory) then this is a fail safe and thus I believe hard coded is just fine.

Reply:
Right, I understand when it's returned.  My question is instead of hard coding would it be possible to grab the settings default value?  Another words grab from the other place it's hard coded so we don't have it a second place.

Reply:
Sure we can, I'm just not sure the best way of doing it. Recommendations?

Reply:
no idea :stuck_out_tongue: so I guess proceed :smile:

Reply:
@graywolf336 You can do `RocketChat.models.Settings.findOneById('FileUpload_MaxFileSize').packageValue`

Reply:
Ah derp, I completely forgot about that.